R/plotting.R

```diff
@@ -101,6 +104,20 @@ timeseries_linechart_basic <- function(df) {
       na.rm = TRUE,
       linewidth = 1
     ) +
+    ggplot2::aes(
+      x = year,
+      y = average_revenue_balance,
+      label = lab
+    ) +
+    # geom_label_repel(xlim = c(-Inf, Inf), ylim = c(-Inf, Inf)) +
+    geom_text_repel(
+      xlim = c(NA, NA),
+      ylim = c(NA, NA),
+      nudge_x = 1,
+      direction = "y",
+      hjust = "left",
```

Comment:
Think a slightly smaller font size might be helpful here given some of the longer names like Bournmouth, Christchurch & Poole
```suggestion
      hjust = "left",
      size = 3.2,
```